Source/hydro/riemann_nd.F90

```diff
@@ -60,20 +60,13 @@ subroutine cmpflx(qm, qp, qpd_lo, qpd_hi, &
     integer, intent(in) :: s_lo(3), s_hi(3)
 
     integer, intent(in) :: idir
-    ! note: ilo, ihi, jlo, jhi are not necessarily the limits of the
-    ! valid (no ghost cells) domain, but could be hi+1 in some
-    ! dimensions.  We rely on the caller to specific the interfaces
-    ! over which to solve the Riemann problems
-    integer, intent(in) :: ilo, ihi, jlo, jhi, kc, kflux, k3d
+    ! note: lo, hi necessarily the limits of the valid (no ghost
```

Comment:
This comment may become confusing later because we're already on a trend towards it being a general rule that the (lo, hi) to loop over are not necessarily the valid domain.